store/interfaces.json

```diff
@@ -0,0 +1,14 @@
+{
+  "default-shelf": {
```

Comment:
I'm not sure about this name... What do you think @klzns ?

Reply:
Do we need this as a block? I mean, `list-context.product-list` does basically the same thing. 

I imagine there will be another block using the Shelf component adding more features. So I would just add a block for this other one.

Reply:
@klzns I'm not sure if `list-context.product-list` does the same thing.. 🤔
This component was created to handle lists of recommended products that would be passed by props, while `list-context.product-list` always searches for products based on other props. I just kept the same behavior for the case of not receiving products..

Reply:
@thalytafabrine 

How would one use this block?

```json

{
  "recommendation": {
    "props": {
      "foo": "bar"
    },
    "children": [
      "rich-text",
      "default-shelf"
    ]
  },
  "default-shelf": {
    "props": {
      "products": [
        {
          "id": "1",
          "name": "Foobar"
        }
      ]
    }
  }
}
```

Reply:
yes, it can be used that way, but our idea is to use it within a `RecommendationShelf` that would make a query for the recommendation API and send the query products to the `default-shelf`